.github/workflows/Release-Dev-Auto.yml

```diff
@@ -83,4 +84,4 @@ jobs:
             Update from the WebUI or click [here](https://github.com/${{ github.repository }}/releases/tag/v${{ steps.semver.outputs.version_tag }}) to download!
             
             **Changelog**:
-            ${{ env.CHANGELOG }}
\ No newline at end of file
+            ${{ env.CHANGELOG }}
```

Comment:
why did this change?

Reply:
Was an accident, my IDE did it on it's own. I tried to look at most of the diffs